src/views/ResultsPage/ResultsListItem.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import * as queryString from 'query-string';
+import { useDispatch } from 'react-redux';
```

Comment:
There is still redux in the ResListItem - I see that it is related to the title caching 
We need to either change requirements of the ticket to say it only removes useSelector part of redux and NOT all of the redux or combine https://github.com/NCIOCPL/clinical-trials-search-app/issues/403 with part of this https://github.com/NCIOCPL/clinical-trials-search-app/issues/447

Reply:
Yeah, thinking a separate commit on this PR might be the way to go. @arcepaul Thoughts?